tests/Integration/Behaviour/Features/Context/CustomerManagerFeatureContext.php

```diff
@@ -272,8 +272,6 @@ public function assertCustomerWasNotFound($customerReference)
         } catch (CustomerNotFoundException $e) {
             return;
         }
-
-        throw new \Exception('Customer exists');
```

Comment:
So the test fails when the exception is not caught I get that this piece of code is useless But then so is the `return` in the catch block

Alternatively, you could use a `$caughtException = null;` before the try/catch block and fill it with caught exception If `$caughtException` is null the test fails (and you can throw a RuntimeException or a NoExceptionAlthoughExpectedException)

Reply:
Done